task5-import-to-s3/task.md

```diff
@@ -0,0 +1,60 @@
+# Task 5 (AWS Simple Storage Service - S3)
+
+**After Lecture 5 "Amazon Simple Storage Service
+(Amazon S3)"**
+
+## Prerequisites
+---
+
+- The task is a continuation of Homework 4 and should be done in the same repos
+- Install the latest version of AWS SDK (https://aws.amazon.com/ru/sdk-for-node-js/)
+- Install the CSV parser package (https://www.npmjs.com/package/csv-parser)
+- Sign in into AWS Console and create a new S3 bucket with a folder, called uploaded
+- Pull latest changes from nodejs-aws-fe (https://github.com/rolling-scopes-school/nodejs-aws-fe)
+
+## TASK 5.1
+---
+
+Create a new service, called **import-service**, with a its own **serverless.yml** file.
```

Comment:
At the same level as product service

your backend project structure should look like 

src/product-service
src/import-service

Reply:
done